src/roslyn/src/NuGet/Microsoft.Net.BuildServerUtils/Microsoft.Net.BuildServerUtils.csproj

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetRoslynSourceBuild)</TargetFrameworks>
+
+    <!-- NuGet -->
+    <IsPackable>true</IsPackable>
+    <IsSourcePackage>true</IsSourcePackage>
+    <PackageId>Microsoft.Net.BuildServerUtils</PackageId>
```

Comment:
I don't have a strong opinion here, but it seems like most of the ecosystem uses `NET`. Having said that, I totally see compilers toolset uses `Net`.

Reply:
I take this back. We are much more inconsistent than I first thought. ignore me.